androidshared/src/main/java/org/odk/collect/androidshared/OneSignTextWatcher.kt

```diff
@@ -0,0 +1,27 @@
+package org.odk.collect.androidshared
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.EditText
+
+class OneSignTextWatcher(private val editText: EditText) : TextWatcher {
```

Comment:
Hah as I was going through the commits I thought to myself "we could definitely extract a `TextWatcher` to share that logic" and then bam there it is. Great minds 🧠